src/cartridges/int_adyen_SFRA/cartridge/adyen/utils/adyenHelper.js

```diff
@@ -749,7 +749,7 @@ let adyenHelperObj = {
   },
 
   // get the fraction digits based on the currency code used to convert amounts of currency for the Adyen Checkout API
-  getFractionDigits(currencyCode) {
+  getFractionDigits(currencyCode = session.currency.currencyCode) {
```

Comment:
the default value assigned to a param should be a constant value. we could move this to a condition when function is called